### DISCUSSION TITLE
[API Request] getAlbumInfo

### DISCUSSION THREAD

**Discussion by sentriz** [opensubsonic/open-subsonic-api#63] [API requests]
### Type of change

API extension

### Proposal description

i propose a getAlbumInfo, akin to getArtistInfo

most servers base getArtistInfo on lastfm's [artist.getInfo](https://www.last.fm/api/show/artist.getInfo) call
getAlbumInfo is a natural extention of that using lastfm's [album.getInfo](https://www.last.fm/api/show/album.getInfo)

take for example how nice an artist page can look on an artist page with airsonic-refix:

![Screen Shot 2023-11-03 at 23 57 59](https://github.com/opensubsonic/open-subsonic-api/assets/6832539/6a83eccd-cfb1-42a0-a66b-0a99894975da)

now take an album page as an example:
![Screen Shot 2023-11-03 at 23 58 22](https://github.com/opensubsonic/open-subsonic-api/assets/6832539/e3c13d7c-ed06-4f8d-8953-3392cf8d6618)

it looks quite bare in comparison

if we add getAlbumInfo, we could really add a lot of interesting bits to album pages like we have for artists like

- wiki
- playcounts
- listeners
- musicbrainz link
- top tracks

![image](https://github.com/opensubsonic/open-subsonic-api/assets/6832539/9fdc4171-03f6-407c-a76d-c70aac4ec203)

the resposne format could be the same getArtistInfo, for example [(taking getArtistInfo for reference)](https://opensubsonic.netlify.app/docs/endpoints/getartistinfo2/)

also pinging @tamland, to see what you think of the idea :grin: 

### Backward compatibility

- [X] No backward compatibility impact.

### API details

`GET /rest/getAlbumInfo.view?id=<album id>`

```json
{
  "subsonic-response": {
    "status": "ok",
    "version": "1.16.1",
    "type": "AwesomeServerName",
    "serverVersion": "0.1.3 (tag)",
    "openSubsonic": true
    "albumInfo": {
        "wiki": "abc abc abc abc abc abc abc",
        "musicBrainzId": "abc",
    }
  }
}
```

### Security impacts

n/a

### Potential issues

it's no different to getArtistInfo as far as I can see

**Comment by dweymouth** [comment]
Uhh.. how is this different from http://www.subsonic.org/pages/api.jsp#getAlbumInfo2? Of course we can extend the payload returned by this if we want to

**Comment by sentriz** [comment]
haha that's hilarious, no idea how I missed that. thanks !
